src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -235,7 +228,7 @@ public LogUnitServer(ServerContext serverContext) {
             // support sparse files.  If we use the default 2GB file size, then
             // every time that a sparse file is closed, the OS will always
             // write 2GB of data to disk.  {sadpanda}  Use this static class
-            // var to signal to RollingLog to use a smaller file size.
+            // var to signal to StreamLogFiles to use a smaller file size.
             maxLogFileSize = 4_000_000;
         }
 
```

Comment:
@slfritchie The line numbers might be offset, but If I understood @dahliamalkhi correctly, I think she is referring to the clean up logic in reset:
https://github.com/CorfuDB/CorfuDB/blob/master/src/main/java/org/corfudb/infrastructure/LogUnitServer.java#L255